infra/modules/terraform-backend-s3/main.tf

```diff
@@ -1,9 +1,10 @@
 data "aws_caller_identity" "current" {}
 data "aws_region" "current" {}
 data "aws_partition" "current" {}
+data "aws_iam_account_alias" "current" {}
 
 locals {
-  tf_state_bucket_name = "${var.project_name}-${data.aws_caller_identity.current.account_id}-${data.aws_region.current.name}-tf-state"
+  tf_state_bucket_name = "${var.project_name}-${data.aws_iam_account_alias.current.account_alias}-${data.aws_region.current.name}-tf-state"
   tf_logs_bucket_name  = "${var.project_name}-${data.aws_caller_identity.current.account_id}-${data.aws_region.current.name}-tf-logs"
```

Comment:
Should this one get the same change?

Reply:
I originally changed it, but the issue specifies to update the name of the bucket where tf files are stored. Should I ask Loren?